docs/data/charts/axis/AxisTextCustomization.js

```diff
@@ -37,7 +42,12 @@ export default function AxisTextCustomization() {
                 },
                 tickLabelStyle: {
                   angle: props.angle,
-                  textAnchor: props.textAnchor,
+                  ...(props.textAnchor === 'unset'
```

Comment:
Having `'unset'` on the selector made me think that the prop now accepts `'unset'`. 

Should we move the selector to be `'undefined'` instead? Which is a real keyword, so less likely to create confusion?

Reply:
Sure thing 👍